arch/arm/src/armv7-m/gnu/arm_exception.S

```diff
@@ -197,7 +197,8 @@ exception_common:
 	 * here prohibits nested interrupts without some additional logic!
 	 */
 
-	setintstack	r2, r3
+	setintstack	r2, r3				/* SP = IRQ stack top */
+	sub		sp, #8					/* Adjust SP (minus 8-byte offset) */
```

Comment:
why sub 8?

Reply:
The same reason as what I commented on arch/arm/src/armv7-m/gnu/arm_lazyexception.S

Reply:
OK, I will remove the line.

Reply:
Done